featurebyte/models/feature.py

```diff
@@ -588,6 +589,11 @@ def _add_tile_derived_attributes(cls, values: dict[str, Any]) -> dict[str, Any]:
             # add a try except block here for the old features that may trigger StopIteration,
             # in this case, we will not add tile related attributes
             return values
+        except Exception:
+            # print a traceback for debugging purpose
+            # without this, the error message will be swallowed by the root_validator
+            print(traceback.format_exc())
+            raise
```

Comment:
We cannot `featurebyte.logging` here as there are circular dependencies issue. Therefore, print statement is used here.

Reply:
Does the added validation cause this or they are not related and this is for something else? Curious what errors were raised here.

Reply:
Before adding the validation, we cannot save the feature due to pydantic model validation error. From the error details, it only shows assertion error. It takes me a while to identify the issue happens inside these statements.